src/bin/ch-remote.rs

```diff
@@ -949,13 +959,15 @@ fn send_migration_data(
     downtime: u64,
     migration_timeout: u64,
     connections: NonZeroU32,
+    tls_dir: Option<&PathBuf>,
```

Comment:
Ahhh, since we take ownership of the data here anyway, we could also use this. Sorry.

```suggestion
    tls_dir: Option<PathBuf>
```

Reply:
Fixed.